javascripts/helpers/util.js

```diff
@@ -1,49 +1,52 @@
 const printToDom = (stringToPrint, elementId) => {
-  const selectedDiv = document.getElementById(elementId);
-  selectedDiv.innerHTML = stringToPrint;
+	const selectedDiv = document.getElementById(elementId);
+	selectedDiv.innerHTML = stringToPrint;
 };
 
-const msgInput = document.getElementById("message-input");
-const navUserSelect = document.getElementById("user-selected");
+const msgInput = document.getElementById('message-input');
+const navUserSelect = document.getElementById('user-selected');
 
 // Generate unique ID for our messages
 const getUniqueId = () => {
-  return Math.random()
-    .toString(36)
-    .substr(2, 16);
+	return Math.random().toString(36).substr(2, 16);
 };
 
 const getTime = () => {
-  return moment().format("h:mm:ss a");
+	return moment().format('h:mm:ss a');
 };
 
 const inputValidation = (messageError, userError) => {
-  if (messageError) {
-    msgInput.classList.add("is-invalid");
-    msgInput.setAttribute("placeholder", "Message field must not be empty");
-  }
-  if (userError) {
-    navUserSelect.classList.replace("btn-success", "btn-danger");
-  }
+	if (messageError) {
+		msgInput.classList.add('is-invalid');
+		msgInput.setAttribute('placeholder', 'Message field must not be empty');
+	}
+	if (userError) {
+		navUserSelect.classList.replace('btn-success', 'btn-danger');
+	}
 };
 
 const resetMessageInput = () => {
-  msgInput.value = "";
+	msgInput.value = '';
 };
 
 // pass in an event and the id of the button as a string
 const disableBtn = (e, buttonId) => {
-  const button = document.getElementById(buttonId);
-  if (e.target.id === buttonId) {
-    button.setAttribute("disabled", "disabled");
-    button.classList.add("disabled");
-  }
+	const button = document.getElementById(buttonId);
+	if (e.target.id === buttonId) {
+		button.setAttribute('disabled', 'disabled');
+		button.classList.add('disabled');
+	}
 };
 
 // pass id of button as a string
-const enableBtn = (buttonId)=> {
-  document.getElementById(buttonId).removeAttribute("disabled");
-  document.getElementById(buttonId).classList.remove("disabled");
+const enableBtn = (buttonId) => {
+	document.getElementById(buttonId).removeAttribute('disabled');
+	document.getElementById(buttonId).classList.remove('disabled');
 };
 
-export { printToDom, getTime, getUniqueId, inputValidation, resetMessageInput, enableBtn, disableBtn };
\ No newline at end of file
+const setScrolDown = (divId) => {
+	const element = document.getElementById(divId).scrollIntoView();
+	// element.scrollTop = element.scrollHeight;
+};
+
```

Comment:
@mjleavell This code is supposed to scroll the last message on screen instead of continuing to push to the bottom.  There was a string interpolation missing but I saw Mohammad have it there.  Must have CTRL-Z one too many times?  Should be good now.